js/includes/alert.js

```diff
@@ -14,7 +14,19 @@
  */
 function addAlert( message ) {
 	// Add the tribe-marketing-alert class to the body tag so we can style appropriately
-	document.body.setAttribute( 'class', document.body.getAttribute( 'class' ) + ' tribe-marketing-alert' )
+	document.body.setAttribute( 'class', document.body.getAttribute( 'class' ) + ' tribe-alert' )
 	// Add the actual message
 	$navbar.after( message )
-}
\ No newline at end of file
+}
+
+/* Remove this when the time comes */
+addAlert( `
+	<div id="tribe-alert">
```

Comment:
It may have followed this pattern in the past, but it won't look good 'as is'. I would recommend we use the same ID as the example on line 6:

```suggestion
	<div id="tribe-marketing-alert">
```